docs/pico_sdk.md

```diff
@@ -63,21 +66,28 @@ In order to use the RF24 libraries in your RP2040 based project:
 
    Alternatively you can add the RF24* repositories as [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
 2. Include their CMakeLists.txt files from the RF24 libraries in your project's top-level
-   CMakeLists.txt file (usually located in the "src" directory). The following snippet
-   assumes that your project's "src" directory is on the same level as the previously
-   mentioned "lib" directory.
+   CMakeLists.txt file (usually located in the `src` directory). The following snippet
+   assumes that your project's `src` directory is on the same level as the previously
+   mentioned `lib` directory.
```

Comment:
Same objection here about "src" and "lib".

Reply:
Sure, no problem. Changed.